config/core/rootfs-images.yaml

```diff
@@ -159,6 +159,14 @@ file_systems:
     ramdisk: bullseye-vdso/20230623.0/{arch}/rootfs.cpio.gz
     root_type: ramdisk
     type: debian
+  debian_bullseye-wifi_nfs:
+    boot_protocol: tftp
+    nfs: bullseye-wifi/20240109.0/{arch}/full.rootfs.tar.xz
```

Comment:
One note: the name of the image should be updated once a bullseye-wifi rootfs image has been built and is available in the KernelCI storage.
I see some staging jobs being run in LAVA (https://lava.collabora.dev/scheduler/alljobs?length=25&search=wifi#table), but they're all failing as such image does not exist yet.

Reply:
Looks like bullseye-wifi rootfs image is available from [2024.01.29](http://storage.kernelci.org/images/rootfs/debian/bullseye-wifi/20240129.0/)